deploy/azuredeploy.json

```diff
@@ -63,14 +69,35 @@
     "postCallSurveyOneQuestionPollAnswerPlaceholder": "",
     "postCallSurveyOneQuestionPollSaveButtonText": "Continue",
     "cosmosDbAccountName": "[concat('db-account-', uniqueString(parameters('appServiceName')))]",
-    "cosmosDbName": "[concat('cosmos-db-', uniqueString(parameters('appServiceName')))]"
+    "cosmosDbName": "[concat('cosmos-db-', uniqueString(parameters('appServiceName')))]",
+    "cognitiveServiceName": "[parameters('cognitiveServiceName')]",
+    "cognitiveServiceId": "[resourceId('Microsoft.CognitiveServices/accounts', parameters('cognitiveServiceName'))]",
+    "cognitiveServicesUserRoleId": "[subscriptionResourceId('Microsoft.Authorization/roleDefinitions', 'a97b65f3-24c7-4388-baec-2e87135dc908')]"
   },
   "resources": [
+    {
+      "type": "Microsoft.CognitiveServices/accounts",
+      "apiVersion": "2023-05-01",
```

Comment:
These are some old looking api versions :S

Reply:
yeah they don't release new versions too often. the next one is a preview